src/server/req_runjob.c

```diff
@@ -264,6 +264,26 @@ clear_from_defr(int sd)
 }
 
 /**
+ * @brief	Wrapper function that calls process_hooks()
+ *
+ * @see		req_runjob()
+ *
+ * @return	int
+ *
+ */
+int
+call_to_process_hooks(struct batch_request *preq, char *hook_msg, size_t msg_len,
+	void	(*pyinter_func)(void))
+{
+	int tmp_var;
```

Comment:
Please use a more meaningful variable name. For example, `return_val`.

Reply:
Or maybe just something as simple as rc.  It's used elsewhere in the code for this sort of thing.